macaw-ppc/tests/Main.hs

```diff
@@ -1,16 +1,27 @@
+{-# LANGUAGE TypeApplications #-}
 module Main ( main ) where
 
-import System.FilePath.Glob ( namesMatching )
+import           Data.Proxy ( Proxy(..) )
+import           System.FilePath.Glob ( namesMatching )
 import qualified Test.Tasty as T
+import qualified Test.Tasty.Options as TTO
+import qualified Test.Tasty.Runners as TTR
 
 import qualified PPC64Tests as PPC64
 import qualified PPC64InstructionCoverage as PPC64
 
+ingredients :: [TTR.Ingredient]
+ingredients = T.includingOptions [ TTO.Option (Proxy @PPC64.SaveMacaw)
+                                 ] : T.defaultIngredients
+
+
 main :: IO ()
 main = do
-  testFiles <- namesMatching "tests/ppc/*.s.expected"
+  testFiles64 <- namesMatching "tests/ppc/*.s.expected"
+  testFiles32 <- namesMatching "tests/ppc32/*.s.expected"
   bins <- namesMatching "tests/ppc/bin/*"
```

Comment:
Is everything under `tests/ppc` related to 64-bit PPC? If so, should it be renamed to `tests/ppc64` for symmetry with the new `tests/ppc32` subdirectory?